src/two_player_collision_example.cpp

```diff
@@ -81,21 +81,19 @@ static constexpr size_t kNumTimeSteps =
 static constexpr float kInterAxleLength = 4.0;  // m
```

Comment:
Yep I'm looking at this now and it's pretty clear that these are only diffs from the `tanmaygautam` branch version, i.e., that branch also has a version of this file (which it shouldn't - so you should delete it from there and remove it from git).

Reply:
I'm currently using Github desktop to manage all the branches and commits. Locally, I've gotten rid of this file from the tanmaygautam branch and pushed the change but I guess that didn't transfer to the remote branch. Do you know where I'm going wrong?

Reply:
hmm. try this: https://stackoverflow.com/questions/8964666/how-do-i-find-a-list-of-files-committed-to-a-branch/18768748

this should let you list all the files in the branch, and then you can see if it's actually there

Reply:
just did this. Doesn't list any of the collision example files in the tanmaygautam branch.

Reply:
I'll try pushing the local branch again.